Mage/Module.cs

```diff
@@ -90,6 +90,11 @@ public class Module<T> : Singleton<T> where T : class, new() {
 	
 	public void setupUsercommands (Action<Exception> cb) {
 		logger.info ("Setting up usercommands");
+
+		if (commands == null) {
```

Comment:
I'm asking, not even suggesting :) Why do they behave differently? If it's null, we don't initialize lists, but when empty we do. Does that make sense? (I'm asking, not rhetorically)